### ISSUE TITLE
Key Error on Creative mode queries on Aragorn (aragorn:v2.3.3)

### ISSUE THREAD

**Issue by YaphetKG** [ranking-agent/aragorn#155]
Olawumi reported a 500 for the query 
```
{
    "message": {
        "query_graph": {
            "edges": {
                "e00": {
                    "subject": "enrich_nodes",
                    "object": "chemical",
                    "predicates": [
                        "biolink:regulates"
                    ],
                    "knowledge_type": "inferred",
                    "qualifier_constraints": [
                        {
                            "qualifier_set": [
                                {
                                    "qualifier_type_id": "biolink:object_direction_qualifier",
                                    "qualifier_value": "downregulated"
                                }
                            ]
                        }
                    ]
                }
            },
            "nodes": {
                "enrich_nodes": {
                    "ids": [
                        "UBERON:0001295"
                    ],
                    "categories": [
                        "biolink:PhysicalEssenceOrOccurrent"
                    ]
                },
                "chemical": {
                    "categories": [
                        "biolink:ChemicalEntity"
                    ]
                }
            }
        }
    },
    "workflow": [
        {
            "id": "lookup"
        }
    ]
}
```

aragorn.renci.org/aragorn fails with  a 500 as it tries to look a matching rule. 

Relevant log 

```
023-02-09 16:21:37,732 | ERROR | common:asyncexecute | 628c424f53fd: Exception '{"predicate": "biolink:regulates", "qualifier_constraints": [{"qualifier_set": [{"qualifier_type_id": "biolink:object_direction_qualifier", "qualifier_value": "downregulated"}]}]}' in execute()
Traceback (most recent call last):
  File "/repo/aragorn/./src/common.py", line 131, in asyncexecute
    query_result, status_code = await entry(message, guid, answer_coalesce_type, caller)
  File "/repo/aragorn/./src/service_aggregator.py", line 146, in entry
    final_answer, status_code = await run_workflow(message, workflow, guid)
  File "/repo/aragorn/./src/service_aggregator.py", line 822, in run_workflow
    message, status_code = await operator_function(message, params, guid)
  File "/repo/aragorn/./src/service_aggregator.py", line 484, in lookup
    return await aragorn_lookup(message, params, guid, infer, answer_qnode)
  File "/repo/aragorn/./src/service_aggregator.py", line 499, in aragorn_lookup
    messages = expand_query(input_message, params, guid)
  File "/repo/aragorn/./src/service_aggregator.py", line 626, in expand_query
    for rule_def in AMIE_EXPANSIONS[key]:
KeyError: '{"predicate": "biolink:regulates", "qualifier_constraints": [{"qualifier_set": [{"qualifier_type_id": "biolink:object_direction_qualifier", "qualifier_value": "downregulated"}]}]}'
2023-02-09 16:21:37,734 | INFO | common:sync_query | 628c424f53fd: Sync query returning.
```

**Comment by cbizon** [comment]
I don't think we have a rule for this predicate... but of course we should be returning a 4xx rather than dying.

**Comment by cbizon** [comment]
fixed in 2.3.10a.  Now an inferred query with a predicate we don't have rules for will still run the direct query (which is part of what happens in an inferred query anyway).
